src/com/owncloud/android/authentication/AuthenticatorActivity.java

```diff
@@ -308,6 +310,16 @@ public void onClick(View v) {
         //Log_OC.e(TAG,  "onCreate end");
     }
 
+    @Override
+    public void onBackPressed(){
+        if(backPressed){
+            super.onBackPressed();

```

Comment:
I'm not able to stop the debugger here, the **backPressed** variable is set to true below, but in which situation it is possible enter this method with backPressed true?

Reply:
@davigonz The bug was quite tough to reproduce because it is quite rare(because if back button is clicked once the activity closes) I just added backPressed condition as a backup that in case if the back button is pressed twice the activity should be closed.